File: python/src/tests/t_shmid.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

id = atrshmlog.get_shmid()

print('shmid : ' + str(id) + ' : ')

print (' ')

exit(0)

# end of test

File: perl/src/tests/t_shmid.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$id = Atrshmlog::get_shmid();

print "shmid : $id : \n";

 
print " \n";

exit (0);

# end of main
